app/src/atoms/AtomLayout.re

```diff
@@ -0,0 +1,21 @@
+module Styles = {
+  open Css;
+
+  let left = style([flex(`num(1.0)), display(`flex)]);
+
+  let right =
+    style([flex(`num(1.0)), display(`flex), justifyContent(flexEnd)]);
+
+  let twoColumns = style([display(`flex), flex(`num(1.0))]);
+};
+
+module Container = {
+  module TwoColumns = {
+    [@react.component]
+    let make = (~children) =>
```

Comment:
This excpects a tuple of children if I understand correctly. Maybe we can reflect that in the type system to know at a glance that this is not just a normal `React.element`.